guides/common/assembly_building_cloud-images.adoc

```diff
@@ -4,7 +4,7 @@ include::modules/proc_creating-custom-rhel-images.adoc[leveloffset=+1]
 
 include::modules/proc_configuring-a-host-for-registration.adoc[leveloffset=+1]
 
-include::modules/proc_registering-hosts.adoc[leveloffset=+1]
+// include::modules/proc_registering-hosts.adoc[leveloffset=+1]
```

Comment:
Not sure what to do about this, because the previous procedure mentions registration via Katello CA consumer.

Reply:
> because the previous procedure mentions registration via Katello CA consumer.

We should not mention it at all, Katello CA consumer will be deprecated soon

Reply:
Technically we consider the Katello CA consumer RPM already deprecated, but haven't been very clear in the replacement yet and no timeline of its removal.

Reply:
I'm reworking it.